File: Java/Maths/Find largest between two numbers - defined values.java
class largestNumber_2num_defined {
    public static void main(String args[]){
        int num1 = 5, num2 = 6;
        if(num1 > num2){
            System.out.println("Largest number is: " +  num1);
        }else if(num2 > num1){
            System.out.println("Largest number is: " +  num2);
        }else{
            System.out.println("Largest number not found. Entered numbers are equal.");
        }
    }
}

File: C++/Maths/Find largest between two numbers - defined values.cpp
#include <iostream>
using namespace std;

int main(){
    int num1 = 5, num2 = 6;
    if(num1 > num2){
        cout << "Largest number is: " << num1;
    }else if(num2 > num1){
        cout << "Largest number is: " << num2;
    }else{
        cout << "Largest number not found. Entered numbers are equal.";
    }
}